docs/jupyter-in-polyglot-notebooks.md

```diff
@@ -28,10 +35,15 @@ install.packages('IRkernel')
 IRkernel::installspec() 
 ```
 
-Then restart VS Code from the Anaconda prompt, and run the following command in a notebook cell:
+You might need to restart VSCode.
```

Comment:
In what cases will the user need to restart VS Code?

Reply:
To be able to retrieve the kernelspecs that were newly added or if you add the environment variables new.

Reply:
Could we add this to the explanation?

E.g. "If you installed the new kernelspec or added new environment variables, you will need to restart VS Code"

Reply:
Done!